udata_front/theme/gouvfr/templates/user/base.html

```diff
@@ -1,5 +1,6 @@
 {% extends theme('layouts/1-column.html') %}
 {% from theme('macros/follow.html') import follow_btn with context %}
+{% from theme('macros/organization_name_with_certificate.html') import organization_name_with_certificate %}
 
 {% set bundle = 'user' %}
 
```

Comment:
We have an breadcrumb that appears like a link to `Users`, but without any href (since we don't have a user list page).

Reply:
This is the default from the DSFR. I'm adding a new style for `a` without href. It's grey and not black to make a small difference between this and the current page